<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>731</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>用户名</string>
         </property>
         <property name="buddy">
          <cstring>userNameList</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="userNameList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="userPasswordList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>备注</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="userRemarkList"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>160</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="AddButton">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>添加</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteButton">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDasd">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="new_entry_nemu"/>
    <addaction name="separator"/>
    <addaction name="save_nemu"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuDasd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="save_nemu">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="load_nemu">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="new_folder_nemu">
   <property name="text">
    <string>New folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="new_entry_nemu">
   <property name="text">
    <string>New entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="undo_nemu">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="Redo_nemu">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="cut_nemu">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="copy_nemu">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="paste_nemu">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="rename_nemu">
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="delete_nemu">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="find_nemu">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="find_next_nemu">
   <property name="text">
    <string>Find next</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNewfile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/yanghuan/.designer/backup/Icon/floder1.png</normaloff>C:/Users/yanghuan/.designer/backup/Icon/floder1.png</iconset>
   </property>
   <property name="text">
    <string>newfile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSavefile">
   <property name="icon">
    <iconset>
     <normalon>C:/Users/yanghuan/.designer/backup/Icon/f13.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>savefile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userNameList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedTheUserNameListItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userPasswordList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedThePasswordListItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userRemarkList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedTheRemarkListItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickedTheUserNameListItem()</slot>
  <slot>clickedThePasswordListItem()</slot>
  <slot>clickedTheRemarkListItem()</slot>
 </slots>
</ui>
